Tutorials/BoseHubbard1d/bosehubbard1d.py

```diff
@@ -12,67 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import netket as nk
+from mpi4py import MPI
 
-from __future__ import print_function
-import json
+# 1D Periodic Lattice
+g = nk.graph.Hypercube(length=12, ndim=1, pbc=True)
```

Comment:
just check again that this tutorials work maybe? now for example it's `n_dim` not `ndim`... PEP!

Reply:
I'll change these over!